Specs/NumberWithUnit/French/CurrencyModel.json

```diff
@@ -888,7 +896,7 @@
   "NotSupportedByDesign": "python",
   "Results": [
     {
-      "Text": "12-cent",
```

Comment:
Is 12-cent a form that typically happens in written French? Formal or informal?
If a common form, we should try to support it too.